great_expectations/data_context/cloud_constants.py

```diff
@@ -7,9 +7,13 @@
 
 
 class GXCloudEnvironmentVariable(str, Enum):
-    BASE_URL = "GE_CLOUD_BASE_URL"
-    ORGANIZATION_ID = "GE_CLOUD_ORGANIZATION_ID"
-    ACCESS_TOKEN = "GE_CLOUD_ACCESS_TOKEN"
+    BASE_URL = "GX_CLOUD_BASE_URL"
+    ORGANIZATION_ID = "GX_CLOUD_ORGANIZATION_ID"
+    ACCESS_TOKEN = "GX_CLOUD_ACCESS_TOKEN"
+    # Deprecated as of 0.15.37
+    _BASE_URL = "GE_CLOUD_BASE_URL"
+    _ORGANIZATION_ID = "GE_CLOUD_ORGANIZATION_ID"
+    _ACCESS_TOKEN = "GE_CLOUD_ACCESS_TOKEN"
```

Comment:
Since env vars are relevant even outside of GX, we should be very particular about the relation to our lib here - the `GX` prefix allows us to do this and once that's set, we can just use "cloud" instead of "ge_cloud" or "gx_cloud".

Reply:
Environment variables, being broader in scope (system level), should be prefixed with GX